catroid/src/main/java/org/catrobat/catroid/formulaeditor/common/FormulaElementOperations.kt

```diff
@@ -109,6 +109,15 @@ object FormulaElementOperations {
     @JvmStatic
     fun getLookName(lookData: LookData?) = lookData?.name ?: ""
 
+    @JvmStatic
+    fun getNumberOfLooks(lookData: LookData?, lookDataList: List<LookData>): Int {
+        return if (lookData?.webRequest == true) {
```

Comment:
" == true" is redundant with the new name of the Getter function.
```suggestion
        return if (lookData?.isWebRequest) {
```

Reply:
the problem here is that if i leave the == true out it wants a not null assertion and i thought we should avoid them